cmd/photobak/main.go

```diff
@@ -33,6 +34,7 @@ var (
 func init() {
 	flag.StringVar(&repoDir, "repo", repoDir, "The directory in which to store the downloaded media")
 	flag.BoolVar(&keepEverything, "everything", keepEverything, "Whether to store all metadata returned by API for each item")
+	flag.BoolVar(&skipRepoChecks, "skiprepochecks", skipRepoChecks, "Don't check items that already exist in the database")
```

Comment:
Do you think consistency checking should be on or off by default? I'm OK with changing the default setting.

I added the integrity checks to help catch errors with the disk or (accidental) editing of files in a repo that should be read-only. If this is uncommon, perhaps we should make it off by default and allow the paranoid or extra careful users to turn it on, just like how `prune` commands are typically run less frequently than standard backups.

If we make consistency checks off by default, let's rename this flag/variable to `integrity-check` or `check-repo` or `verify-checksums` or something.